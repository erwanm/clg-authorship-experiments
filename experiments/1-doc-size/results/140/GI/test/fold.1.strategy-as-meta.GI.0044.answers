1903-us-tjoas 1854-hbs-smofl	0.350526
1895-wta-alae 1906-mt-cfma	0.200064
1891-fmc-ttf 1881-haj-fcbtp	0.000316
1872-haj-ptf 1906-rwc-ttolp	0.475476
1898-hj-ttm 1879-hj-h	0.550046
1874-wta-tcw 1900-haj-adoh	0.00091
1878-hj-te 1882-lma-ajsb-v6	6.2e-05
1883-hj-rh 1898-hj-itc	0.600137
1885-hj-gr 1910-us-sts	0.150139
1914-us-sm 1897-ewaoc-tdof	0.500017
1896-espw-cfal 1887-espw-jtf	0.099982
1887-espw-tgb 1900-espw-wtg	0.200335
1894-cdw-tgh 1907-fmc-tdr	0.300291
1900-fmc-itpotk 1905-fmc-fm	1
1889-fmc-g 1898-haj-tyb	0.000501
1890-hj-ttm 1904-hj-tgb	0.9503
1878-hbs-pp 1870-hbs-tbc	1
1901-us-km 1921-us-tbol	0.400293
1883-hj-rh-v2 1872-wta-lb	0.000214
1866-haj-ccc-ia 1871-hbs-mwai	0.250131
1880-hj-c 1885-hj-altof	0.150163
1898-fmc-ari-v2 1893-fmc-kl-v2	0.000496
1918-rwc-tlg 1863-wta-isao	0.700038
1884-ab-sbts 1898-us-apom	0.000126
1889-cfw-jl 1915-rwc-p	0.375161
1872-mt-ri 1895-haj-tdw	0.000926
1902-fmc-c 1883-fmc-tl	1.2e-05
1859-hbs-tmw 1887-fmc-pp	0.100218
1884-wta-Sac 1882-wta-aa	0.900086
1871-haj-tt 1876-wta-vao	0.800111
1913-es-tm 1911-rwc-tgr	0.200372
1905-es-tpoe 1907-es-tlotg	0.050423
1899-haj-mmv 1866-haj-hf	0.825455
1891-fmc-do 1876-cdw-itl	0.950198
1892-fmc-pg 1883-fmc-dc	0.550047
1917-ab-bn 1922-us-tcmc	0.300314
1889-cdw-sths 1869-espw-mwag	3.9e-05
1902-hj-twotd 1896-hj-tsop	1
1888-haj-bb 1899-haj-ra	0.600045
1913-ew-tcotc 1868-haj-rd	0.450049
1868-espw-tga 1919-rwc-tsos	0.049989
1900-haj-ainy 1919-ew-m	0.000893
1881-mt-tpatp 1899-haj-aoatb	0.000707
1891-fmc-k 1893-fmc-tr	0.700105
1881-hj-tpoal 1890-fmc-acmr	0.900133
1869-haj-mtmb 1903-haj-bba-ph	0.450443
1885-fmc-atoalp 1905-fmc-wso	0.000117
1868-wta-mob 1870-wta-faf	0.100338
1871-hbs-pawt 1904-rwc-isotu	0.100711
1875-cfw-cn 1864-lma-m	0.400029
1914-mh+-mhce-nf 1908-fmc-tp	0.000168
1893-fmc-md 1895-fmc-ajs	0.100322
1864-wta-tsb 1883-wta-sh	1
1900-rwc-taoamm 1910-rwc-tgm	1
1903-fmc-thor 1887-fmc-s	0.000434
1884-mt-taohf 1874-mt-lotm	0.900452
1907-espw-tldup 1871-espw-tsp	0.300074
1866-espw-gcj 1882-haj-bn	1
1883-haj-tbb 1888-es-ttatv	0.925147
1908-fmc-s 1908-fmc-tws	0.800111
1889-wta-wtel 1911-rwc-tcl	0.000419
1904-rwc-sm 1886-hj-pc	0.600443
1918-us-jh 1897-ew-tdof	0.000465
1872-lma-ss 1867-lma-tagomtt	0.550143
1910-ew-tomag 1889-es-atg	0.500242
1889-espw-tsfi 1895-espw-tsojc	1
1905-haj-fftf 1870-haj-rar	0.600198
1882-espw-dz 1909-mt-isd	0.000225
1907-ew-tfott 1896-hj-toh	0.000727
1868-wta-dad 1885-wta-sts	0.500326
1909-rwc-tdm 1897-ab-tdohy	0.725473
1891-haj-tetb 1868-haj-faf	0.999963
1893-haj-dtn 1866-haj-tcw	1
1869-wta-dtr 1865-wta-waw	0.500426
1891-cdw-oi 1873-hbs-wish	1
1879-hj-dm 1903-hj-ta	0.15042
1874-cfw-tac 1882-fmc-mi	0.95005
1893-espw-dm 1896-ab-boat	0.001052
1896-espw-tsasa 1894-espw-asl	0.500148
1879-haj-tym 1869-haj-pjb	1
1920-us-1tsoap 1913-us-dg	0.000413
1917-es-ow-sh 1873-hbs-pl	0.099851
1855-wta-aa 1873-wta-tyc	1
1879-haj-ttb 1894-fmc-wg	5.6e-05
1898-fmc-vc 1895-fmc-t	1
1867-espw-gyatgc 1901-espw-a	0.20007
1896-fmc-c 1885-fmc-zamc	0.800145
1855-hbs-tmf 1866-espw-gb	0.650068
1897-fmc-aroy 1884-fmc-aap	0.000321
1883-espw-btg 1911-hj-to	0.400151
1862-hbs-tpooi 1875-hbs-waon	0.750159
1883-fmc-ars 1885-fmc-tcotk	0.500066
1886-hbs-tsw 1854-hbs-smofl-v2	1
1903-ew-ivatg 1878-cdw-hikab	0.199984
1858-wta-pap 1897-haj-fof	1
1902-hj-twotd-v2 1888-hj-tr	0.85035
1871-hbs-ofs 1888-fmc-si	0.000512
1907-ab-rm 1901-hj-tsf	0.0001
1882-haj-ffbts 1893-haj-ianwatgfoa	0.000303
1908-rwc-tfl 1910-rwc-ap	1
