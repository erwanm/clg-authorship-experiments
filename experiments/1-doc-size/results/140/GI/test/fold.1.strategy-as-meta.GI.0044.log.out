apply-config.sh: applying config '/tmp/expe1/140/process//selected-meta-configs/fold.1.strategy-as-meta.GI.0044' to cases '/home/moreaue/dump/expe1//results/140/GI/test/only-cases.tmp.TBPWwtF1'; results written to '/home/moreaue/dump/expe1//results/140/GI/test/fold.1.strategy-as-meta.GI.0044'
apply-config.sh: done.
